src/components/CytoscapeLayout/CytoscapeLayout.tsx

```diff
@@ -118,13 +97,47 @@ export default class CytoscapeLayout extends React.Component<CytoscapeLayoutProp
         this.handleMouseOut(cytoscapeEvent);
       }
     });
+
+    // when the graph is fully populated and ready, we need to perform additional things
     this.cy.ready((evt: any) => {
       if (!this.props.isReady) {
+        // Don't allow a large zoom if the graph has a few nodes (nodes would look too big).
+        if (this.cy.zoom() > 2.5) {
```

Comment:
Should this be a constant perhaps in the `config` object?

Reply:
This isn't my code - it came from another PR that tragically I had to move around because of git conflicts. So its not really for this PR, but its here because of git conflict resolution. I told you people to stop editing these files while I am in here :) See PR #298